src/index.ts

```diff
@@ -4,19 +4,13 @@ import { createI18n as create } from 'i18n-mini';
 import type { I18nOptions } from 'i18n-mini';
 import type { I18n } from './types';
 
+export type { I18n, I18nOptions };
+
 export { defineMessages } from 'i18n-mini';
 export type { I18nPresets } from 'i18n-mini';
 
-export { I18nProvider } from './components/I18nProvider';
-export { Text } from './components/Text';
-export { Numeric } from './components/Numeric';
-export { DateTime } from './components/DateTime';
-export { useI18n } from './context';
```

Comment:
I don't think we should delete this line.

Reply:
Yep, rebased and readded the line, totally my mistake!